qubes/storage/reflink.py

```diff
@@ -352,17 +383,14 @@ def size(self):
 
     @property
     def usage(self):
-        ''' Return volume disk usage from the VM's perspective. It is
-            usually much lower from the host's perspective due to CoW.
-        '''
```

Comment:
Any reason to remove this comment?

Reply:
See [commit](https://github.com/QubesOS/qubes-core-admin/pull/449/commits/f482916999ee4d2868281a9a5d07e340a1d9185d) message